geospaas_harvesting/crawlers.py

```diff
@@ -231,12 +250,44 @@ def _get_links(cls, html):
         return parser.links
 
 
+class PODAACCrawler(WebDirectoryCrawler):
+    """
+    Crawler for harvesting the data of PODAAC
+    """
+    LOGGER = logging.getLogger(__name__ + '.PODAACCrawler')
+    FOLDERS_SUFFIXES = ('/contents.html',)
+    FILES_SUFFIXES = ('.nc', '.nc.gz')
+    EXCLUDE = ['?']
+
+    def get_download_url(self, resource_url):
+        return resource_url
+
+
+class OpenDAPCrawler(WebDirectoryCrawler):
+    """
+    Crawler for harvesting the data which are provided by OpenDAP
+    """
+    LOGGER = logging.getLogger(__name__ + '.OpenDAPCrawler')
+    FOLDERS_SUFFIXES = ('/catalog.html',)
+    FILES_SUFFIXES = ('.nc',)
+    # We exclude "EASE-Grid map projections" and "southern hemispheres" from harvesting provess
+    EXCLUDE = ['/thredds/', 'http', '_sh_polstere', 'ease', ]
```

Comment:
According to the issue notes: https://github.com/nansencenter/django-geo-spaas-harvesting/issues/16#issue-653052772 and my questions from @akorosov,  there is no need to collect the data of south pole! We only need the north pole! Thus, the '_sh_polstere' is excluded. 

'ease' is the projection of data that we don't want it. We only want 'polstere' projection. Thus, the 'ease' is excluded. 
Please prove this @akorosov  once again because these datasets are never harvested if we merge this code!

Reply:
Then this should not be hard-coded. This kind of distinction varies with the usage, so it should be a parameter of the crawler that can be set in the configuration file and passed to the crawler through the harvester.

Reply:
I agree

Reply:
Done in the next commit